lib/raygun.rb

```diff
@@ -146,13 +146,19 @@ def track_exception_sync(exception_instance, env, user, retry_count)
       end
 
       if retry_count > 0
-        new_exception = e.exception("raygun4ruby encountered an exception processing your exception")
-        new_exception.set_backtrace(e.backtrace)
 
         env[:custom_data] ||= {}
         env[:custom_data].merge!(original_stacktrace: exception_instance.backtrace)
 
-        track_exception(new_exception, env, user, retry_count - 1)
+        ::Raygun::Breadcrumbs::Store.replace([])
+
+        ::Raygun.record_breadcrumb(
+          :message => "Payload modified:\n\n We have stripped the breadcrumb information as this payload exceeded maximum payload size of 128KB. \n",
```

Comment:
Would this always be accurate? I'm concerned that any failure would now look like an issue due to an entity-oversized which may not be the case?

Reply:
You are correct, it may not be the case

Reply:
Do you want me to update the message?